<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>805</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dendron GUI</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>801</width>
      <height>551</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <property name="sizeConstraint">
      <enum>QLayout::SetMaximumSize</enum>
     </property>
     <property name="topMargin">
      <number>5</number>
     </property>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <property name="leftMargin">
        <number>10</number>
       </property>
       <property name="rightMargin">
        <number>10</number>
       </property>
       <item>
        <widget class="QLabel" name="label">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Devices</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="devicesComboBox"/>
       </item>
       <item>
        <widget class="QPushButton" name="searchButton">
         <property name="text">
          <string>Search</string>
         </property>
         <property name="icon">
          <iconset resource="iconos.qrc">
           <normaloff>:/icons/search</normaloff>:/icons/search</iconset>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="connectButton">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Connect</string>
         </property>
         <property name="icon">
          <iconset resource="iconos.qrc">
           <normaloff>:/icons/connect</normaloff>:/icons/connect</iconset>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="startButton">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Start</string>
         </property>
         <property name="icon">
          <iconset resource="iconos.qrc">
           <normaloff>:/icons/start</normaloff>:/icons/start</iconset>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="markButton">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Mark</string>
         </property>
         <property name="icon">
          <iconset resource="iconos.qrc">
           <normaloff>:/icons/mark</normaloff>:/icons/mark</iconset>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="recordButton">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Record</string>
         </property>
         <property name="icon">
          <iconset resource="iconos.qrc">
           <normaloff>:/icons/save</normaloff>:/icons/save</iconset>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="OSCButton">
         <property name="text">
          <string>OSC</string>
         </property>
         <property name="icon">
          <iconset resource="iconos.qrc">
           <normaloff>:/icons/osc</normaloff>:/icons/osc</iconset>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>805</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_File">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="actionConfiguration"/>
    <addaction name="action_Quit"/>
   </widget>
   <widget class="QMenu" name="menuA_bout">
    <property name="title">
     <string>&amp;Help</string>
    </property>
    <addaction name="actionA_bout"/>
   </widget>
   <addaction name="menu_File"/>
   <addaction name="menuA_bout"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="action_Quit">
   <property name="text">
    <string>&amp;Quit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionA_bout">
   <property name="text">
    <string>A&amp;bout</string>
   </property>
  </action>
  <action name="actionConfiguration">
   <property name="text">
    <string>&amp;Configuration</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="iconos.qrc"/>
 </resources>
 <connections/>
</ui>
